### ISSUE TITLE
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36

### ISSUE THREAD

**Issue by rickiey** [filecoin-project/lotus#8394]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.15.1-rc3+calibnet+git.003df89ed
```


### Describe the Bug

when I enable FVM , run `lotus daemon` , sync error:

2022-03-29T16:15:08.325 INFO filcrypto::fvm::machine > fil_create_fvm_machine: start
2022-03-29T16:15:08.325 INFO filcrypto::fvm::machine > using FVM V1
2022-03-29T16:15:08.325 INFO filecoin_proofs::api::winning_post > verify_winning_post:finish
2022-03-29T16:15:08.325 INFO filcrypto::proofs::api > verify_winning_post: finish
2022-03-29T16:15:08.334 INFO filecoin_proofs::api::winning_post > verify_winning_post:finish
2022-03-29T16:15:08.334 INFO filcrypto::proofs::api > verify_winning_post: finish
2022-03-29T16:15:08.349 INFO filecoin_proofs::api::winning_post > verify_winning_post:finish
2022-03-29T16:15:08.349 INFO filcrypto::proofs::api > verify_winning_post: finish
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:thread '<unnamed>' panicked at '36
source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36
250:36
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36
thread '<unnamed>' panicked at 'source slice length (2195990) does not match destination slice length (64)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/multihash-0.16.1/src/hasher_impl.rs:250:36

### Logging Information

```text
caused by: [bafy2bzacecv5did35f7pgvkn2azmbofjumhevvsqbbnd6ieg2xkkeaylhb5lc] 2 errors occurred:                                                                                                    [5/1956]
        * block had invalid messages:                                                                                                                                                                                                   
    github.com/filecoin-project/lotus/chain/consensus/filcns.(*FilecoinEC).ValidateBlock.func1                                                                                                                                          
        /root/workplace/lotus/chain/consensus/filcns/filecoin.go:134                                                                                                                                                                    
  - failed to compute tipsettate for {bafy2bzaceafbejwwu3wsv7r2nr2ih7rpofnfosngt6m7tmk4du5jv5neigpe6,bafy2bzaceaif3sj2hdk2bhord23yholztdpsa5kiih3ox7b343mshxc6nwxmu,bafy2bzacecdlm76h2tqdbh7xwzkvolyg2r76bag4ao46u3lcdthaoaisti6rw,bafy2bzaced3qlkmd2jhla4jav56yrcxvbz62bofg6c274gubylw5pnuiqmsca,bafy2bzacec5lie2jx6xfsdpiwa2ovazffkdywsiuctd5whxye7ggttecdwy7s}:                                                                                                              
    github.com/filecoin-project/lotus/chain/consensus/filcns.(*FilecoinEC).checkBlockMessages
        /root/workplace/lotus/chain/consensus/filcns/filecoin.go:461
  - making vm:
    github.com/filecoin-project/lotus/chain/consensus/filcns.(*TipSetExecutor).ApplyBlocks
        /root/workplace/lotus/chain/consensus/filcns/compute_state.go:189
  - Rust panic: no unwind information:
    github.com/filecoin-project/filecoin-ffi.CreateFVM
        /root/workplace/lotus/extern/filecoin-ffi/fvm.go:75
        * get tipsetstate(816042, [bafy2bzaceafbejwwu3wsv7r2nr2ih7rpofnfosngt6m7tmk4du5jv5neigpe6 bafy2bzaceaif3sj2hdk2bhord23yholztdpsa5kiih3ox7b343mshxc6nwxmu bafy2bzacecdlm76h2tqdbh7xwzkvolyg2r76bag4ao46u3lcdthaoaisti6rw bafy2bzaced3qlkmd2jhla4jav56yrcxvbz62bofg6c274gubylw5pnuiqmsca bafy2bzacec5lie2jx6xfsdpiwa2ovazffkdywsiuctd5whxye7ggttecdwy7s]) failed:
    github.com/filecoin-project/lotus/chain/consensus/filcns.(*FilecoinEC).ValidateBlock.func4
        /root/workplace/lotus/chain/consensus/filcns/filecoin.go:170
  - making vm:
    github.com/filecoin-project/lotus/chain/consensus/filcns.(*TipSetExecutor).ApplyBlocks
        /root/workplace/lotus/chain/consensus/filcns/compute_state.go:189
  - Rust panic: no unwind information:
    github.com/filecoin-project/filecoin-ffi.CreateFVM
        /root/workplace/lotus/extern/filecoin-ffi/fvm.go:75

)
```


### Repo Steps

1 git checkout tags/v1.15.1-rc3 -b v1.15.1-rc3
2. make calibnet
3. export LOTUS_USE_FVM_EXPERIMENTAL=1 
4. lotus daemon

**Comment by TippyFlitsUK** [comment]
Hey @rickiey 

Thank you for your report!

This issue will be resolved in the upcoming `1.15.1-rc4` release. 

Please set `LOTUS_USE_FVM_EXPERIMENTAL=0` for the time being.

Many thanks

**Comment by TippyFlitsUK** [comment]
Hi @rickiey 

[1.15.1-rc4](https://github.com/filecoin-project/lotus/releases/tag/v1.15.1-rc4) is now available. Please upgrade when possible and confirm if the error is resolved for you.

Many thanks!
